packages/service/src/domain/commands/processors/playback-current-command.ts

```diff
@@ -16,32 +16,36 @@ class PlaybackCurrentCommand extends CommandProcessor {
   }
 
   async execute(command: Command): Promise<CommandProcessingResponse> {
-    const includeQueue = command.rawArgs === 'queue';
+    const includeQueue = command.rawArgs?.includes('queue') ?? false;
+    const includePreview = command.rawArgs?.includes('embed') ?? false;
 
     const playerStateRequest: PlayerStateRequestEvent = { id: 'PlayerStateRequestEvent' };
     const currentPlayerState: PlayerState = await this.playerEventStream.sendToPrimaryPlayer(playerStateRequest);
 
     const text = this.buildMessage(
-      currentPlayerState.currentlyPlaying?.song ?? null,
+      currentPlayerState.currentlyPlaying ?? null,
       currentPlayerState.queue ?? [],
       includeQueue,
+      includePreview,
     );
 
     return new CommandProcessingResponseBuilder()
       .fromMarkdown(text)
       .build();
   }
 
-  private buildMessage(current: (Song | null), queue: Song[], includeQueue: Boolean): string {
```

Comment:
We should handle `current` is `null` early. What we're doing right now may lead to for example wrong URL being generated (`https://www.youtube.com/watch?v=null`).